demo/src/main/java/App.java

```diff
@@ -31,6 +31,21 @@ public static void main(String[] args) {
         );
         Pyroscope.setStaticLabels(mapOf("region", "us-east-1"));
 
+        Thread.sleep(5 * 60 * 1000);
+
+        // This is a naive implementation for the stop method
+        PyroscopeAgent.stop(
+            new PyroscopeAgent.Options.Builder(
+                new Config.Builder()
+                    .setApplicationName("demo.app{qweqwe=asdasd}")
+                    .setServerAddress("http://localhost:4040")
+                    .setFormat(Format.JFR)
+                    .setLogLevel(Logger.Level.DEBUG)
+                    .setLabels(mapOf("user", "tolyan"))
+                    .build())
+                .build()
+        );
```

Comment:
This code will start a second profiler and stop it immediately

Reply:
Thanks for your review @simonswine , it makes sense to add AtomicReference<Options> and stop everything that was set up above. I will let you know if I run into anything

Reply:
I also noticed again when I ran this test, it starts the profiler and then waits for sometime with Thread.sleep and stops it but the start thread is still running so we get an error there saying `Error Dumping profiler`
![Uploading Screenshot 2023-10-19 at 6.40.13 AM.png…]()

Reply:
@simonswine pls re-review. Thanks